target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -296,6 +296,11 @@ tplink,tl-mr6400-v1)
 	ucidef_set_led_netdev "wan" "WAN" "tp-link:white:wan" "eth1"
 	ucidef_set_led_netdev "4g" "4G" "tp-link:white:4g" "usb0"
 	;;
+tplink,tl-wr841hp-v3)
+	ucidef_set_netdev "wan" "WAN Data" "tp-link:green:wan" "eth1" "tx rx"
+	ucidef_set_netdev "wan_red" "WAN Link" "tp-link:red:wan" "eth1" "link"
```

Comment:
That seems different from what the manual says; in accordance with most other TP-Link devices, I'd just use green wan LED here.

Reply:
If you do that, you should be able to just add this device to the case with tplink,archer-c58-v1.